src/commands/createNewProject/createNewProject.ts

```diff
@@ -45,7 +46,8 @@ export async function createNewProject(
             const languagePicks: QuickPickItem[] = [
                 { label: ProjectLanguage.JavaScript, description: '' },
                 { label: ProjectLanguage.CSharp, description: '' },
-                { label: ProjectLanguage.Java, description: '' }
+                { label: ProjectLanguage.Java, description: '' },
+                { label: ProjectLanguage.Python, description: '(Preview)' }
```

Comment:
Please remove 'Python' from this quick pick until we have full support and we know we're ready to release